### PR TITLE
Feature/issue 17 page header

### PR DESCRIPTION
**Pull request by DanielvanVliet**
~~Currently the Icons are missing due to missing components in the component library.~~
I added a placeholder Icon from MUI icons, using a custom Icon did not integrate very well with our MUI based components. To fix this I added an issue for [adding icons to the component library](https://github.com/Gemeente-DenHaag/denhaag-component-library/issues/46) instead. When those are added we can replace this placeholder.

Besides this, the `<MenuItem>`component has a missing property, see issue 19 on the nlds-react-components repo. Until this is merged we can use a ts-ignore.

### CODE REVIEW

**Review by ogjtech — DISMISSED** [review]

**Comment by DanielvanVliet** [comment]
Ready for review once again

**Review by GewoonMaarten — APPROVED** [review]

**Review by joostvanviegen — APPROVED** [review]

**Review by bddjong — APPROVED** [review]

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { Button } from '@gemeente-denhaag/button/Button'
```
Comment (mjcarsjens) [inline src/components/Header/index.tsx:2]: Why is this import not with the other components, and why is this one using a named import while the others all use the default import?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -14,10 +14,16 @@ import Stepper from '@gemeente-denhaag/stepper/Stepper'
 import Step from '@gemeente-denhaag/step/Step'
 import StepLabel from '@gemeente-denhaag/steplabel/StepLabel'
 import StepContent from '@gemeente-denhaag/stepcontent/StepContent'
+import AppBar from '@gemeente-denhaag/appbar/AppBar'
+
+import Header from '@/components/Header'
```
Comment (mjcarsjens) [inline src/pages/index.tsx:19]: Why is this a relative import instead one from `@gemeente-denhaag/header`?

File: src/components/Header/index.tsx (hunk 1 above)
Comment (DanielvanVliet) [reply src/components/Header/index.tsx]: I've made that import consistent with the others.

**Review by GewoonMaarten — APPROVED** [review]

**Pull request merged**